### DISCUSSION TITLE
gore sites blocked

### DISCUSSION THREAD

**Discussion by jproffer** [StevenBlack/hosts#1969] [General]
I use the "Unified Hosts" version of this list - no additions.  

Noticed while removing a redirect site from my hosts file in order to visit some links - there's a bunch of gore and other non-tracker, non-ad sites are automatically included in this file (including domains with no subdomains).  Any reason why?  This has me questioning the validity of all the other domains.

**Comment by StevenBlack** [comment]
Hi John @jproffer. Can you give me specific examples?
